warehouse/templates/email/removed-project-release-file/subject.txt

```diff
@@ -0,0 +1,17 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block subject %}{% trans file=file %}The file {{ file }} has been deleted.{% endtrans %}{% endblock %}
```

Comment:
I think we might want to make this a more generic subject. Many projects have lots of files for a given release, and as is, this would generate a unique email thread for every deleted file, which might be noisy.

Maybe something like "File deleted for {{ project }} {{ release.version }}"?